documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -0,0 +1,654 @@
+|                                |                                               |
+|    str                         |    as(<symbol>, str)                          |
++--------------------------------+-----------------------------------------------+
+
+Numerical operations
+^^^^^^^^^^^^^^^^^^^^
++----------------------------------+-----------------------------------------------+
+| Python                           | Dylan                                         |
++==================================+===============================================+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    isinstance(obj,               |    instance?(obj, <number>)                   |
+|      (int, float, complex))      |                                               |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    isinstance(obj, complex)      |    instance?(obj, <complex>)                  |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    isinstance(obj,               |    instance?(obj, <real>)                     |
+|      (int, float))               |                                               |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    isinstance(obj, int) or       |    instance?(obj, <rational>)                 |
+|      (isinstance(obj, float) and |                                               |
+|        obj == int(obj))          |                                               |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    isinstance(obj, int)          |    instance?(obj, <integer>)                  |
+|                                  |    integral?(num)                             |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    n1 = n2                       |    n1 = n2                                    |
```

Comment:
You want `==` on the Python side.  And probably should delete `=` on the Dylan side since in this context we're comparing numbers only.

Reply:
My fault in left side, but I don't understand why the right side is wrong.